app/models/crime.rb

```diff
@@ -1,5 +1,13 @@
 class Crime < ApplicationRecord
   has_one :skillset
   belongs_to :boss, class_name: 'User'
-  belongs_to :right_hand, class_name: 'User'
+  belongs_to :right_hand, class_name: 'User', optional: true
+
+  validates_associated :skillset
+
+  validates :title, length: { minimum: 5 }, allow_blank: false
```

Comment:
I think we shouldn't have length validation on the title. Some people might want to use acronyms for their title, which can be less than 5 characters :)

Reply:
Then maybe two chars min?